src/gam.py

```diff
@@ -7413,6 +7416,9 @@ def doUpdateVaultMatter(action=None):
     action_kwargs = {}
   if body:
     print u'Updating matter %s...' % sys.argv[3]
+    # bah, API requires name to be sent on update even when it's not changing
+    result = callGAPI(v.matters(), u'get', matterId=matterId, view=u'BASIC')
```

Comment:
if body['name'] is already set we shouldn't do an expensive API call, make this conditional on name not being set.